.github/workflows/quarkus.yml

```diff
@@ -37,8 +37,19 @@ env:
 
 jobs:
   build-mandrel:
-    name: Mandrel build
+    name: Mandrel build - ${{ matrix.jdk }}
     runs-on: ubuntu-18.04
+    strategy:
+      fail-fast: false
+      matrix:
+        jdk: ['OpenJDK-latest', 'OpenJDK-ea']
+        include:
+          - jdk: 'OpenJDK-latest'
+            tag_cmd: "curl https://api.github.com/repos/AdoptOpenJDK/openjdk11-upstream-binaries/releases/latest | jq -r '.tag_name'"
+            tag_suffix: ""
+          - jdk: 'OpenJDK-ea'
```

Comment:
I'd go with `OpenJDK-11-latest-ea` and `OpenJDK-11-latest-ga`. They're both latest, just different release types.